share/completions/emaint.fish

```diff
@@ -0,0 +1,32 @@
+## Global Opts
+complete -c emaint -s h -l help         -d "show this help message and exit"
+complete -c emaint -s c -l check        -d "Check for problems (a default option for most modules)"
+complete -c emaint      -l version      -d "show program's version number and exit"
+complete -c emaint -s f -l fix          -d "Attempt to fix problems (a default option for most modules)"
```

Comment:
"Attempt to fix problems (default for most modules)"?

Reply:
Don't ask me, I just copypaste messages.

Reply:
https://gitweb.gentoo.org/proj/portage.git/tree/pym/portage/emaint/defaults.py#n6

Reply:
That was a suggestion for improvement.